### PR TITLE
Dutch: A few strings I found last minute

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: config/locales/simple_form.nl.yml [hunk 1]
```diff
@@ -4,13 +4,17 @@ nl:
     hints:
       defaults:
         avatar: PNG, GIF of JPG. Maximaal 2MB. Wordt teruggeschaald naar 120x120px
+        digest: Wordt na een lange periode van inactiviteit verzonden, met een samenvatting van vermeldingen tijdens je afwezigheid.
         display_name: Maximaal 30 tekens
```
Comment (Gargron) [inline config/locales/simple_form.nl.yml]: Cannot have value + children, remove value

**Review by Gargron — CHANGES_REQUESTED** [review]

File: config/locales/simple_form.nl.yml (hunk 1 above)
Comment (jeroenpraat) [reply config/locales/simple_form.nl.yml]: I don't understand @Gargron? This comes directly from https://github.com/tootsuite/mastodon/blob/33ea042decc03c494c78a896934455ef79abc16b/config/locales/simple_form.en.yml#L7

The only difference is see the comma and the period.

Comment (Gargron) [reply config/locales/simple_form.nl.yml]: Umm no the comment is about line 8

```yml
key: This is incorrect
  child: foo
  another: bar
```

Comment (jeroenpraat) [reply config/locales/simple_form.nl.yml]: oooooh :-( Sorry, fixing it now

**Review by Gargron — APPROVED** [review]

**Pull request merged**